src/main/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandler.java

```diff
@@ -18,6 +18,7 @@
 import org.folio.inventory.dataimport.handlers.actions.CreateHoldingEventHandler;
 import org.folio.inventory.dataimport.handlers.actions.CreateInstanceEventHandler;
 import org.folio.inventory.dataimport.handlers.actions.CreateItemEventHandler;
+import org.folio.inventory.dataimport.handlers.actions.CreateMarcAuthoritiesEventHandler;
```

Comment:
```suggestion
import org.folio.inventory.dataimport.handlers.actions.CreateMarcAuthorityEventHandler;
```
Maybe will be better to use singular?